migrations/20240810070920_create_chat.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+create table if not exists chats (
+    id bigserial primary key,
```

Comment:
https://stackoverflow.com/questions/4848964/difference-between-text-and-varchar-character-varying